source/_components/alexa.markdown

```diff
@@ -9,17 +9,50 @@ sharing: true
 footer: true
 logo: amazon-echo.png
 ha_category: Voice
-featured: false
+featured: true
```

Comment:
Look at this page: https://home-assistant.io/components/

One can only be added if we either remove 1 or add 4 more. You can remove forecast.io being featured.